File: CS101/Replit/LengthMost.py
def count(phrase):
    words = phrase.split()
    word_lengths = {}
    
    for word in words:
        length = len(word)
        if length not in word_lengths:
            word_lengths[length] = 0
        word_lengths[length] += 1
    
    max_count = max(word_lengths.values())
    max_lengths = [length for length, count in word_lengths.items() if count == max_count]
    max_lengths.sort()
    
    return f"{max_count}:{'-'.join(map(str, max_lengths))}"

File: CS201/Replit/LengthMost.java
import java.util.*;

public class LengthMost {
	public String count(String phrase) {
		String[] words = phrase.split(" ");
		Map<Integer, Integer> wordLengthCounts = new HashMap<>();
		int maxCount = 0;
		for (String word : words) {
			int length = word.length();
			wordLengthCounts.put(length, wordLengthCounts.getOrDefault(length, 0) + 1);
			maxCount = Math.max(maxCount, wordLengthCounts.get(length));
		}

		List<Integer> maxWordLengths = new ArrayList<>();
		for (Map.Entry<Integer, Integer> entry : wordLengthCounts.entrySet()) {
			if (entry.getValue() == maxCount) {
				maxWordLengths.add(entry.getKey());
			}
		}
		Collections.sort(maxWordLengths);

		StringBuilder result = new StringBuilder();
		result.append(maxCount).append(":");
		for (int i = 0; i < maxWordLengths.size(); i++) {
			result.append(maxWordLengths.get(i));
			if (i < maxWordLengths.size() - 1) {
				result.append("-");
			}
		}
		return result.toString();
	}
}